Integrations/CarbonBlackProtect/CarbonBlackProtect.yml

```diff
@@ -132,8 +193,8 @@ script:
         is any valid field in the object that is being queried. Operator (: LIKE,
         ! NOT LIKE, < Less than, > Greater than, + logical AND, - logical OR, | separating
         values) is any of valid operators (see below). All operators consist of a
-        single character. Value is compared with an operator and depends on field type.
-        See more: https://developer.carbonblack.com/reference/enterprise-protection/8.0/rest-api/#searching'
+        single character. Value is compared with an operator and depends on field
+        type. See more: https://developer.carbonblack.com/reference/enterprise-protection/8.0/rest-api/#searching'
       isArray: false
       name: query
       required: false
```

Comment:
line 203 still has "int23" in it

Reply:
Weird I was sure I removed it... good catch